Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -527,7 +528,12 @@ public ParserState GetOrCreateModuleState(IVBComponent component)
 
         public ParserState GetModuleState(IVBComponent component)
         {
-            return _moduleStates.GetOrAdd(new QualifiedModuleName(component), new ModuleState(ParserState.Pending)).State;
+            return GetModuleState(new QualifiedModuleName(component));
+        }
+
+        public ParserState GetModuleState(QualifiedModuleName module)
+        {
+            return _moduleStates.GetOrAdd(module, new ModuleState(ParserState.Pending)).State;  //Why does getting the module state potentially create a new one? That is not too intuitive.
```

Comment:
the comment isn't helpful. also the answer should be obvious:

`// an unknown module's state was requested. The only reasonable answer is <tt>Pending</tt>. We need to also update the dictionary`